packages/host/app/routes/card.gts

```diff
@@ -63,7 +41,7 @@ export default class RenderCard extends Route<Model | null> {
       : new URL('./', ownRealmURL);
 
     try {
-      let cardResource = getCard(this, () => url.href);
+      let cardResource = getCard(this, () => url.href); //TODO: this method creates a resource entangled with the route lifetime, but that is wrong since route instances live forever AND we are not using this in a resource-y way
```

Comment:
this might actually be used in a resource-y way, but i think its too soon to tell--specifically if we want to adjust the live updating status for this card (e.g. go from a static card to a live card or vice versa). A lot of that will depend on what we want to do in  guest mode (which i believe is what this model drives). Right now we'll get a live updating card in guest mode--unsure if that is really what we want.....

Reply:
I've removed the TODO for now. It was meant to be a temporary breadcrumb for Ed and I while working on an unrelated issue. Thanks for the background.